### ISSUE TITLE
Looks like site was hacked; now hosts spam content

### ISSUE THREAD

**Issue by mLuby** [SandersForPresident/RideWithBernie#32]
https://www.bernie2016events.org/ (linked from App's [Find a Rideshare] button) has got a bunch of spammy posts that suggest to me it's been taken over by hackers.

**Comment by mLuby** [comment]
Tagging you all hoping someone has the access to handle this: 
- @kyleschmolze
- @cmuld3r
- @jasnow
- @buddhistsforbernie

**Comment by jasnow** [comment]
This website is a wordpress website. Ran "whois" and got this: https://whois.icann.org/en/lookup?name=www.bernie2016events.org

**Comment by kyleschmolze** [comment]
Ouch.

Should we take down RideWithBernie, disable the button, add a warning message?

**Comment by jasnow** [comment]
What is the relationship between www.bernie2106events.org website 
and RideWithBernie? I do not see a button to run https://ridewithbernie.com ?

**Comment by mLuby** [comment]
@jasnow this is the relationship; in looking for a live demo I clicked the link ridewithbernie.com.

<img width="700" alt="screen shot 2019-02-22 at 5 27 08 pm" src="https://user-images.githubusercontent.com/2483420/53276963-30679f80-36c7-11e9-9613-688c00027369.png">

Clicking [Find a Rideshare] takes you to https://bernie2106events.org.
<img width="646" alt="screen shot 2019-02-22 at 5 28 23 pm" src="https://user-images.githubusercontent.com/2483420/53276996-55f4a900-36c7-11e9-9e87-da2bb9482726.png">

**Comment by kyleschmolze** [comment]
Yeah @jasnow we used to redirect users to bernie2016events.com, because you could click any event and there was a rideshare button on it that took you back to our site. But looks like he gave up that domain and someone put a parking page on it :( The red button flow still works, but I doubt anyone is using it. Happy to comment out the green button part until we have another bernie event site to integrate with :)
